### PR TITLE
Profiles userflow

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,3 @@
-FROM praekeltfoundation/molo-bootstrap:5.4.6-onbuild
```
Comment (codiebeulaine) [inline Dockerfile]: please don't remove this

File: requirements.txt [hunk 2]
```diff
@@ -11,4 +11,3 @@ gunicorn
 django_compressor==2.0
 celery<4.0
 django-google-analytics-app==2.1.6
-Unidecode==0.04.16
```
Comment (codiebeulaine) [inline requirements.txt]: why are we removing this?

**Review by projectmushroom — APPROVED** [review]
👍

File: requirements.txt (hunk 2 above)
Comment (Mitso) [reply requirements.txt]: @codiebeulaine I had reverted to this running lower version and re-testing.  
Adding all reverts back.

**Pull request merged**